pkg/webhook/csidriver/csidriver.go

```diff
@@ -0,0 +1,101 @@
+package csidriver
+
+import (
+	"net/http"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+	admissionctl "sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+// VolumeSourceType represents a volume source type
+type VolumeSourceType string
+
+const (
+	WebhookName         string           = "sharedresourcecsidriver-csidriver"
```

Comment:
This is weird spacing, did your code editor do this?

Reply:
yes it is aligning it with what is below